jwst/straylight/straylight_step.py

```diff
@@ -14,7 +14,7 @@ class StraylightStep (Step):
     def process(self, input):
 
         # Open the input data model
-        with datamodels.ImageModel(input) as input_model:
+        with datamodels.IFUImageModel(input) as input_model:
```

Comment:
See here we're specifically loading into an IFUImageModel, as opposed to using the generic datamodels.open(). I think we should do the same in fringe_step.py

Reply:
👍